internal/pkg/terraswap/databases/grpc/classic.grpc.go

```diff
@@ -8,13 +8,13 @@ import (
 
 	"github.com/pkg/errors"
 
+	wasm "github.com/CosmWasm/wasmd/x/wasm/types"
 	"github.com/cosmos/cosmos-sdk/types"
+	ibc "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
```

Comment:
They used v4

Reply:
updated to v6 for mainnet(phoenix) and classic(columbus) 
(FYI.) `v4` and `v6` has same the same message type for DenomTrace